.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
   - pip install six
   # Travis needs to update pip/setuptools
   - pip install --upgrade setuptools pip
-  - pip install .
+  - pip install -e .
```

Comment:
Did this end up being needed in the end? I'd prefer to keep it a non-dev install if possible.

Reply:
No, I'm not sure it is. I'll try reverting that and see if it goes OK.